### ISSUE TITLE
[BUG] Characteristic Status is changed to Enumeration if model is imported into Editor

### ISSUE THREAD

**Issue by bs-jokri** [eclipse-esmf/esmf-aspect-model-editor#52]
**Describe the bug**
A model contains a status characteristic. 
If the ttl file of this model is imported into the editor the Status characteristic is changed to Enumeration and the default value is removed.

**Where**
Aspect Model Editor v4.3.0

**Comment by michelu89** [comment]
Hey @bs-jokri,

Could you kindly provide a detailed, step-by-step guide outlining your process, and include any aspect models you've imported?

Regrettably, I am unable to reproduce the issue you're experiencing on my end. There is also a possibility that the issue has been addressed in the latest release, version 4.4.1.

**Comment by bs-jokri** [comment]
Hi @michelu89,

use this model
```
@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
@prefix : <urn:bamm:io.openmanufacturing.digitaltwin:1.0.0#>.

:AspectDefault a bamm:Aspect;
    bamm:properties (:property1);
    bamm:operations ();
    bamm:events ().
:property1 a bamm:Property;
    bamm:characteristic :Characteristic1.
:Characteristic1 a bamm-c:State;
    bamm:dataType xsd:string;
    bamm-c:values ("one" "two");
    bamm-c:defaultValue "one".

```

store it in a file and import it into the editor. In 4.3.0 the characteristic is switched from state to enumeration.

Probably it is fixed in 4.4.1 but the CX consortium still needs to stick to 4.3.0 as they need to work on BAMM models as the migration to SAMM is not fully approved and executed so far.

**Comment by michelu89** [comment]
Thank you for providing the example. We have noticed that there's an issue related to the state characteristic too. It's being mistakenly identified as an enumeration during instantiation, leading to improper import. We appreciate your patience and will work to resolve this at the earliest.
